package-lock.json

```diff
@@ -3327,24 +3380,6 @@
         "@types/node": "*"
       }
     },
-    "@types/graphql-upload": {
-      "version": "8.0.4",
-      "resolved": "https://registry.npmjs.org/@types/graphql-upload/-/graphql-upload-8.0.4.tgz",
-      "integrity": "sha512-0TRyJD2o8vbkmJF8InppFcPVcXKk+Rvlg/xvpHBIndSJYpmDWfmtx/ZAtl4f3jR2vfarpTqYgj8MZuJssSoU7Q==",
-      "requires": {
-        "@types/express": "*",
-        "@types/fs-capacitor": "*",
-        "@types/koa": "*",
-        "graphql": "^15.3.0"
-      },
-      "dependencies": {
-        "graphql": {
-          "version": "15.4.0",
-          "resolved": "https://registry.npmjs.org/graphql/-/graphql-15.4.0.tgz",
-          "integrity": "sha512-EB3zgGchcabbsU9cFe1j+yxdzKQKAbGUWRb13DsrsMN1yyfmmIq+2+L5MqVWcDCE4V89R5AyUOi7sMOGxdsYtA=="
-        }
-      }
-    },
```

Comment:
AS 2.21 replaced both `graphql-upload` and `@types/graphql-upload` with a fork that has types built in, so that's probably what's causing this?

Reply:
(I think the fork isn't being added in this PR because it was already in package-lock because there were some uses of AS 2.21 and some uses of older AS in package-lock already? Or just that npm hadn't quite deduped yet?)